samples/bot-type-ahead-search-adaptive-cards/csharp/README.md

```diff
@@ -0,0 +1,94 @@
+---
+page_type: sample
+description: Demonstrating the feature of type ahead search (static and dynamic)control on Adaptive Cards.
+products:
+- office-teams
+- office
+- office-365
+languages:
+- C#
+extensions:
+contentType: samples
+createdDate: "11-11-2021 23:30:17"
+---
+# Type ahead search control on Adaptive Cards
+
+This sample shows the feature of type ahead search (static and dynamic) control on Adaptive Cards.
```

Comment:
We did not mentioned that how are we going to get these static or dynamic cards??